.github/workflows/cypress.yml

```diff
@@ -9,4 +9,6 @@ jobs:
       - name: Install dependencies
         run: npm install
       - name: Component tests
-        run: npm run test:ct
\ No newline at end of file
+        run: npm run test:ct
```

Comment:
Two things I'm wondering here. 

1.) This only seems to run the Cypress tests, but do we not want to run bot `jest` and `cypress` test suites, to later combine the reports?

2.) The `nyc` docs suggest that it should be trivial to set up and only require prepending what ever test runner you have with `nyc ...`. Maybe we do not need the babel stuff at all and can try and follow that?[0] and have here as a `run` just `nyc --reporter=lcov npm run test`.


[0] https://github.com/istanbuljs/nyc/blob/master/docs/setup-codecov.md

Reply:
I'm not sure we should experiment with the NYC in this PR the goal was to get the data about test coverage.
It is made based on the example from vulnerability and ocp advisor, so we're already using the same solution in 2 more repos

Reply:
@bastilian I'm not sure what we are waiting for. The PR is working, codecov.io have the correct report added. Merge?

Reply:
@Fewwy I have no objections to merge it, it was just a comment/question. But I might not be the best to give an approval, since I only skimmed not reviewed the PR. ;)

Reply:
@bastilian It's all good! Thanks for the information, that made me think through a couple of things again :)
@gkarat Need your approval